did.js

```diff
@@ -5,34 +5,22 @@
 
 import {DidStore} from 'bedrock-web-did-store';
 import {LDKeyPair} from 'crypto-ld';
-import didv1 from 'did-veres-one';
+import v1 from 'did-veres-one';
 
-const {VeresOneDidDoc} = didv1;
+const veresDriver = v1.driver({mode: 'test'});
 
-export async function generateDid() {
+export async function generateInvokeKey(keyType) {
```

Comment:
This is not specific to the key purpose, so the method name shouldn't include it. Also, we should change the TODO to a FIXME and change `bedrock-web-kms` to `web-kms-client`.

Reply:
Thanks, done.